java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/OFBooleanValue.java

```diff
@@ -81,13 +79,6 @@ public void writeTo(ChannelBuffer bb) {
         bb.writeByte(getInt());
     }
 
-    private static class Reader implements OFMessageReader<OFBooleanValue> {
```

Comment:
There should be a public final static field READEr that is the (singleton) instance of this... Feel free to add it.